sherpa/astro/io/crates_backend.py

```diff
@@ -441,8 +431,6 @@ def read_table_blocks(arg, make_copy=False):
         for name in names:
             cols[ii][name] = crate.get_column(name).values
 
-    if close_dataset:
-        close_crate_dataset(dataset)
```

Comment:
Why do we not need this any longer? I'm probably missing something obvious, but usually I try to avoid touching anything crate related with a twn-foot-pole...

Reply:
In crates there was a memory leak/problem where the solution was to add the snip method to the dataset. `close_crate_dataset` then called this method, if it existed. I can't remember what version of CIAO removed this work around, but it was a while ago...

Aha - CIAO 4.5 removed the need for it (it's at the end of the https://cxc.cfa.harvard.edu/ciao/releasenotes/ciao_4.5_release.html#DataModel right before chips). It looks like we never officially documented when it was taken out.